crates/sui-framework/packages/sui-system/sources/sui_system_state_inner.move

```diff
@@ -1121,11 +1121,18 @@ module sui_system::sui_system_state_inner {
         self.stake_subsidy.get_distribution_counter()
     }
 
+    // NEVER remove `#[test_only]`
     #[test_only]
     public(package) fun set_epoch_for_testing(self: &mut SuiSystemStateInnerV2, epoch_num: u64) {
         self.epoch = epoch_num
     }
 
+    // NEVER remove `#[test_only]`
+    #[test_only]
+    public(package) fun increment_epoch_for_testing(self: &mut SuiSystemStateInnerV2) {
```

Comment:
ditto, are you sure you need `increment` if you have `set_...`?

Reply:
see above